src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -60,6 +35,11 @@ using var meterProvider = Sdk.CreateMeterProviderBuilder()
 Refer to [Program.cs](../../examples/runtime-instrumentation/Program.cs) for a
 complete demo.
 
+Additionally, this examples sets up the OpenTelemetry Prometheus exporter, which
+requires adding the package
+[`OpenTelemetry.Exporter.Prometheus`](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.Prometheus.AspNetCore/README.md)
```

Comment:
use httplistener one as this example seems to be just console app, and not asp.net core

Reply:
I updated the link and the text. However we don't have the `OpenTelemetry.Exporter.Prometheus.HttpListener` NuGet package yet. I'll probably keep the package name until we release a new package later.

Reply:
https://www.nuget.org/packages?q=+OpenTelemetry.Exporter.Prometheus